ansible/roles/ubuntu-worker/tasks/rebar.yml

```diff
@@ -0,0 +1,37 @@
+- name: create rebar group
+  group:
+    name: rebar
+    state: present
+
+- name: create rebar user
+  user:
+    name: rebar
+    createhome: yes
+    group: rebar
+    state: present
+
+- name: download rebar
+  get_url:
+    url: https://github.com/rebar/rebar/archive/{{ rebar_version }}.tar.gz
```

Comment:
Oh, that's good to know. I'll remove the whole rebar stuff. Funny though, both https://github.com/apache/couchdb/blob/master/Dockerfile and https://github.com/klaemo/docker-couchdb/blob/2.0-dev/Dockerfile install rebar separately.